src/panel/pages/events/components/Settings.tsx

```diff
@@ -7,17 +7,17 @@ import {
   faFastForward,
 } from "@fortawesome/free-solid-svg-icons";
 import { Collapsible } from "../../../components";
-import { useTimelineContext } from "../../../context";
+import { useTimelineContext, START_PADDING } from "../../../context";
 
 export const Settings: FC<ComponentProps<typeof Container>> = (props) => {
   const [collapsed, setCollapsed] = useState(true);
   const handleExpandToggle = useCallback(() => setCollapsed((c) => !c), []);
   const { setPosition, startTime } = useTimelineContext();
 
-  const handleBackClick = useCallback(() => setPosition(startTime), [
-    setPosition,
-    startTime,
-  ]);
+  const handleBackClick = useCallback(
+    () => setPosition(startTime - START_PADDING),
```

Comment:
Would this apply the padding twice as [it's also applied here](https://github.com/FormidableLabs/urql-devtools/blob/a29910048b4a2c3a1e0a4382abd59ffade89c74c/src/panel/context/Timeline.tsx#L97)?

Reply:
Good spot - we're safe because we're getting the max result (`Math.max(-100, -100)`)